### ISSUE TITLE
New ports configuration is missing mod_ports.so on CentOS 7

### ISSUE THREAD

**Issue by vgirnet** [sous-chefs/apache2#554]
### Cookbook version
5.2.0

### Chef-client version
Chef: 14.3.37

### Platform Details
CentOS Linux release 7.5.1804 (Core) 

### Scenario:
Generate a web application

### Steps to Reproduce:
Just include latest cookbook apache2 5.2.0 in dependencies, and generate a web site configuration

### Expected Result:
httpd to be running

### Actual Result:

```
  * service[httpd] action restart
    
    ================================================================================
    Error executing action `restart` on resource 'service[httpd]'
    ================================================================================
    
    Mixlib::ShellOut::ShellCommandFailed
    ------------------------------------
    Expected process to exit with [0], but received '1'
    ---- Begin output of /usr/bin/systemctl --system restart httpd ----
    STDOUT: 
    STDERR: Job for httpd.service failed because the control process exited with error code. See "systemctl status httpd.service" and "journalctl -xe" for details.
    ---- End output of /usr/bin/systemctl --system restart httpd ----
    Ran /usr/bin/systemctl --system restart httpd returned 1
    
    Resource Declaration:
    ---------------------
    
     30: service node['apache']['service_name']
    
    Compiled Resource:
    ------------------
    
    service("httpd") do
      action [:nothing]
      default_guard_interpreter :default
      service_name "httpd"
      enabled nil
      running nil
      masked nil
      pattern "httpd"
      declared_type :service
    end
    
    System Info:
    ------------
    chef_version=14.3.37
    platform=centos
    platform_version=7.5.1804
    ruby=ruby 2.5.1p57 (2018-03-29 revision 63029) [x86_64-linux]
    program_name=/opt/chef/embedded/bin/chef-client
    executable=/opt/chef/embedded/bin/chef-client
    
```

When troubleshooting, found this:
```
Aug 30 07:50:14 tk-default-centos-7-2e637e httpd[23412]: httpd: Syntax error on line 112 of /etc/httpd/conf/httpd.conf: Syntax error on line 1 of /etc/httpd/mods-enabled/ports.load: Cannot load /usr/lib64/httpd/modules/mod_ports.so into server: /usr/lib64/httpd/modules/mod_ports.so: cannot open shared object file: No such file or directory
```

```
# ls -la /usr/lib64/httpd/modules/mod_ports.so
ls: cannot access /usr/lib64/httpd/modules/mod_ports.so: No such file or directory
```

**Comment by vgirnet** [comment]
Cookbook version 5.0.1 works just fine.

**Comment by damacus** [comment]
Fixed by #557 👍 which is out on the supermarket now.
